### ISSUE TITLE
Unable to parse request body response from DO when using C#

### ISSUE THREAD

**Issue by euphario** [F5Networks/f5-declarative-onboarding#79]
In C#, System.Net.Http automatically adds "charset=UTF-8" to the Content-Type header; 

```
POST /shared/declarative-onboarding HTTP/1.1
Content-Type: application/json; charset=UTF-8
Authorization: Basic YWRtaW46
X-F5-Auth-Token: RCY7LIBX4LJEZABMQPZTV34PXS
X-Forwarded-For: 192.168.200.1
Referer: 192.168.200.1
```

This causes DO to respond with;

```
HTTP/1.1 400 Bad Request
X-Powered-By: Express
Content-Type: application/json
Content-Length: 442
Pragma: no-cache
Cache-Control: no-store, no-cache, must-revalidate
Expires: -1
Server: restnoded
Date: Thu, 24 Oct 2019 17:02:21 GMT
Connection: keep-alive

{"id":"62b57a65-ad1a-4ac4-8ddb-0427fad81b79","selfLink":"https://localhost/mgmt/shared/declarative-onboarding/task/62b57a65-ad1a-4ac4-8ddb-0427fad81b79","code":400,"status":"ERROR","message":"bad declaration","errors":["Unable to parse request body. Should be JSON format."],"result":{"class":"Result","code":400,"status":"ERROR","message":"bad declaration","errors":["Unable to parse request body. Should be JSON format."]},"declaration":{}}
```

Tested as well in postman. Any declaration sent with the extra charset=UTF-8 fails with DO.

Workaround in C# is to use text/plain for the Content-Type.

This is specific to DO, AS3 does not exhibit this behavior.

**Comment by dstokesf5** [comment]
I have confirmed this bug and have created internal product backlog item AUTOTOOL-901 to track it.

**Comment by dstokesf5** [comment]
This issue has been resolved as part of the 1.13.0 release.
